app/lib/seeder.rb

```diff
@@ -503,6 +503,26 @@ def run
     Fraud::Indicators::Timezone.create(name: "America/New_York", activated_at: DateTime.now)
     Fraud::Indicators::Timezone.create(name: "America/Los_Angeles", activated_at: DateTime.now)
     SearchIndexer.refresh_search_index
+
+    state_file_arizona_intake = StateFileAzIntake.find_or_create_by!(
+      primary_first_name: "Ari",
+      primary_last_name: "Zona",
+      consented_to_terms_and_conditions: "yes",
+      primary_birth_date: 35.years.ago,
+      email_address: "ari.zona@example.com",
+      federal_submission_id: "12345202201011232170",
+      email_address_verified_at: Time.current
+    )
+
+    if state_file_arizona_intake.efile_submissions.none?
+      efile_submission = EfileSubmission.find_or_create_by!(
+        irs_submission_id: "9999992021199yrv4rab",
+        data_source: state_file_arizona_intake,
+      )
+      efile_submission.transition_to!(:preparing)
```

Comment:
Will this cause statesman to transition this anyway? Do we want the system to start polling for acknowledgements for this?

Reply:
the `if Rails.env.production?` at the beginning of the seeder file should prevent this from running in production